host/fs/fs_linux.go

```diff
@@ -5,6 +5,9 @@
 package fs
 
 import "syscall"
+import "golang.org/x/sys/unix"
```

Comment:
Please use `goimports` to update the imports automatically. It'll keep them ordered.
Do not use package unix; periph doesn't have any external imports.

Reply:
yes that's what the other constants are.